examples/webservice-ws-with-resources-config/README_pt.adoc

```diff
@@ -0,0 +1,86 @@
+
+CXF API é reutilizável mas além disso você pode configurar os interceptadores através do `openejb-jar.xml`(localizados na WEB-INF).
+Como um rápido lembrete, a configuração no `openejb-jar.xml`, você pode utilizar uma configuração parecida a esta, com o seu próximo EJB:
+
+[source,xml]
+----
+
+<openejb-jar xmlns="http://www.openejb.org/openejb-jar/1.1">
+  <ejb-deployment ejb-name="CalculatorBean"> <!-- configurar os valores do bean, no resources.xml -->
+    <properties>
+      cxf.jaxws.in-interceptors = wss4j
+    </properties>
+  </ejb-deployment>
+</openejb-jar>
+----
+Para um webservice Pojo isto é o mesmo, mas usando o pojo-deployment em vez do ejb-deployment.
+
+Então uma vez que fez sua seleção de prefixo, e conhece onde é para escrever a configuração, basta utilizar as seguintes entradas:
+
+* *properties*: propriedades da fábrica servidora
+
+* *features*: CXF para características
+
+* *in-interceptors*: CXF dentro do interceptadores
+
+* *out-interceptors*: CXF fora dos interceptadores
+
+* *in-fault-interceptors*: CXF dentro dos interceptadores para tratamento de erros 
+
+* *out-fault-interceptors*: CXF fora dos out interceptadores para tratamento de erros 
+
+* *databinding*: servidor de vinculação de dados
+
+* *providers (somente para JAX-RS endpoint)*: lista de provedores JAX-RS
+
+* *skip-provider-scanning (somente para JAX-RS)*: utilizar ou não um provedor de varredura (por padrão true)
+
+Para características de interceptadores, a regra é o mesma: lista de valores separados por vírgula. Cada valor da lista, é um nome de classe qualificado ou um id de serviço, é um ou o outro, no resources.xml.
+
+== Exemplo para JAX-WS
+Para configurar o WSS4J no EJB `CalculatorBean` para adicionar uma instância no openejb-jar.xml:
+
+[source,xml]
+----
+
+<resources>
+  <!-- os interceptadores -->
+  <Service id="wss4j1" class-name="org.apache.openejb.server.cxf.config.WSS4JInInterceptorFactory" factory-name="create">
+    action = UsernameToken
+    passwordType = PasswordText
+    passwordCallbackClass = org.superbiz.ws.security.PasswordCallbackHandler
+  </Service>
+</resources>
+----
+== passwordCallbackClass
+
+`passwordCallbackClass` é uma propriedade do `WSS4JInInterceptorFactory` e seu valor da classe é `PasswordCallbackHandler`.
+
+`PasswordCallbackHandler` utiliza `org.apache.wss4j.common.ext.WSPasswordCallback` para prover senhas no mecanismo de callback.
+
+
+==== Exemplo
+
+Exemplo completo pode ser encontrado https://github.com/apache/tomee/blob/master/examples/webservice-ws-with-resources-config/src/main/java/org/superbiz/ws/security/PasswordCallbackHandler.java[aqui]:
+
+
+=== Executando os testes
+
+Exemplo de projeto completo pode ser encontrado https://github.com/apache/tomee/tree/master/examples/webservice-ws-with-resources-config[aqui].
+Isoe é um projeto maven, e todos os testes pode ser executados rodando o comando `mvn clean test`.
```

Comment:
```suggestion
Ele é um projeto maven, e todos os testes podem ser executados rodando o comando `mvn clean test`.
```

Reply:
I am make a mistake, writed with error in the beginning sentence, "Isoe é", the correct is "Isso é". But can be "Isto é" or "Este é", but not "Ele é", because "maven" is not a person, this is a thing.

Reply:
For me, using `Isso é`, `Isto é`, `Este é` or `Ele é` it's irrelevant because we translate to Portuguese and we don't have this level of distinction between things or persons. For English you are correct, maven is a thing, not a person and they aren't the same. The important thing is a clear message.

Reply:
The error was mine, i have of the revise the translations more times.

Reply:
Man, you make a great job. This is a collaborative work. We are together to make this work. Without errors. Without fault. Everyone is doing a brilliant job.

Reply:
Is corrected